python/py-pew/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           github 1.0
 PortGroup           select 1.0
 
-revision            1
+revision            2
```

Comment:
As you are just adding a new port here, the py38 one, no revision bump is required.